src/lib/observe.ts

```diff
@@ -30,19 +30,21 @@ interface PerformanceEntriesHandler {
  * try/catch to avoid errors in unsupporting browsers.
  */
 export const observe = (
-  type: string,
+  types: string[],
   callback: PerformanceEntriesHandler,
   opts?: PerformanceObserverInit,
 ): PerformanceObserver | undefined => {
   try {
-    if (PerformanceObserver.supportedEntryTypes.includes(type)) {
+    if (types.every(type => PerformanceObserver.supportedEntryTypes.includes(type))) {
       const po: PerformanceObserver = new PerformanceObserver((list) => {
         callback(list.getEntries());
       });
-      po.observe(Object.assign({
-        type,
-        buffered: true,
-      }, opts || {}) as PerformanceObserverInit);
+      for (let type of types) {
+        po.observe(Object.assign({
+          type,
+          buffered: true,
+        }, opts || {}) as PerformanceObserverInit);
```

Comment:
Right now, the same options are passed for all registered types.  This is fine for this specific case.

But in general we may want to have `[string, PerformanceObserverInit][]` (aka array of tuple) instead